### ISSUE TITLE
It should be possible to view `REVERTcast`s without `Set Printing All`

### ISSUE THREAD

**Issue by JasonGross** [coq/coq#13985]
I suggest `Set Printing Tactic Cast Nodes` or `Set Printing All Cast Nodes`.  Here's an example test-case:
```coq
Set Printing Tactic Cast Nodes.
Goal (fun x => x) True.
  cbv. Show Proof. (* ?Goal *)
  Set Printing All. Show Proof. (* (?Goal : (fun x : Prop => x) True) *)
```

**Comment by ppedrot** [comment]
Strongly related to #13911. At the last call I proposed to reuse that syntax for revert casts.

**Comment by ppedrot** [comment]
This was fixed by #14773.
